UPGRADE.md

```diff
@@ -10,9 +10,19 @@ The `autocompile` option is now the default option for the behavior of `\gregori
 
 ### Gregorio and Grana Padano fonts
 
-The gregorio and granapadano fonts, as well as their "-op" Dominican variants, will now be available as separate downloads.  If you were using these fonts, either download and install them or switch to greciliae, which continues to be distributed along with Gregorio by default.
+The Gregorio and Grana Padano fonts, as well as their "-op" Dominican variants, will now be available as a separate download.  If you were using these fonts, either download `supp_fonts-5_0_0.zip` and install them or switch to Greciliae, which continues to be distributed (with its "-op" Dominican variants) along with Gregorio by default.
```

Comment:
The fact that Gregorio is the name of the font and the name of this application is confusing in this statement.  That's the reason I originally used the font filenames (lowercase) rather than the title-case name of the font.